lib/licensee/project_files/license_file.rb

```diff
@@ -20,6 +20,11 @@ def self.name_score(filename)
         return 0.5 if filename =~ /licen[sc]e/i
         0.0
       end
+
+      # case-insensitive block to determine if the given file is LICENSE.lesser
+      def self.lesser_gpl_score(filename)
+        filename.casecmp('license.lesser').zero? ? 1 : 0
```

Comment:
Fair enough. I'll remove the "or LICENSE.lesser" from http://choosealicense.com/licenses/lgpl-3.0/ which seems to have been there from the beginning of the site https://github.com/github/choosealicense.com/commit/39198c0b320d97abffcdbf912bcd348a15de6577#diff-c84257b1f58e7d12bce47aed7f0746b0R208